src/styles/stars.scss

```diff
@@ -0,0 +1,35 @@
+.card__review-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  .stars {
+    display: flex;
+    align-items: center;
+
+    &--4 {
+      display: flex;
+    }
```

Comment:
The `display: flex;` rule in `.stars--4` is redundant since `.stars` already defines the display as flex. You can safely remove this rule to keep the code clean.